### DISCUSSION TITLE
k3s node not read, cni plugin not initialized

### DISCUSSION THREAD

**Discussion by maciejklimek** [k3s-io/k3s#8803] [General]
Hello, 
I have a problem with getting the server node to start up after k3s installation. 
 `kubectl describe node` gives a clue : 
`container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network plugin returns error: cni plugin not initialized`

Any ideas how to tackle this?

**Comment by brandond** [comment]
Wait for flannel to start. If the message doesn't go away within a few seconds, check for other warnings or errors in the logs. You didn't open this as an issue so we're missing all the info the issue template would normally ask for, but can you confirm that you haven't disabled flannel by setting --flannel-backend=none?

**Reply by maciejklimek** [reply]
I have created an issue and supplied required information. I confirm I have not disabled flannel during the installation.
